skinlist.lua

```diff
@@ -2,7 +2,8 @@ local dbgprint = false and print or function() end
 
 --- @param path     Path to the "textures" directory, without tailing slash.
 --- @param filename Current file name, such as "player.groot.17.png".
-local function process_skin_texture(path, filename)
+--- @return On error: false, error message. On success: true, skin key
+function skins.register_skin(path, filename)
```

Comment:
Will be added to the API documentation in a later PR.

Reply:
I assume list point "3." is OK too? :upside_down_face: 

In either case - thank you for testing. If there's no further issues found, I'll go ahead merging this PR in about 24 hours.

Reply:
I'm not entirely certain about point 3 - that's the hand key ambiguity. I looked at the key again and found it still ambiguous, but I thought that's because this issue wasn't marked addressed in this PR.

My goal is to externalize skins from this mod, so we can go "full upstream", while maintaining our skins in a worldmods folder #31 and having a way to retrofit skins into a running server (currently blocked by the problem that the hand is a new block, which cannot be registered during runtime)